recipes/expansionhunter/meta.yaml

```diff
@@ -1,47 +1,53 @@
 {% set name = "ExpansionHunter" %}
-{% set version = "4.0.2" %}
-{% set sha256 = "cbe66f09b8d86499966ae6da8655de04976e097227656495644fde4a69104525" %}
+{% set version = "5.0.0" %}
+{% set sha256 = "921d1f8be658f2bdd144150278210bb921d605af921816e94a7508c0b9b9dfff" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
-
-build:
-  number: 1
-
 source:
   url: https://github.com/Illumina/ExpansionHunter/archive/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-    - brackets.patch  # [osx]
-
+    - 0001-adjust-cmake-targets.patch
+    - 0002-specify-boost-for-make_unique.patch
+    - 0003-add-missing-limits-import.patch
+    - 0004-add-fmt-formatters-for-classes.patch
+    - 0005-apply-bioconda-4.0.2-patches.patch  # [osx]
+build:
+  number: '0'
+  run_exports:
+    - {{ pin_compatible('abseil-cpp', max_pin='x.x') }}
```

Comment:
Those are usually not needed and packages like boost come with their own run-exports by default.

Reply:
It's interesting, I was having issues with the local tests I was running creating packages that weren't exporting their pins. But it seems like that's actually an issue with indexing the local channels I'm using, and you're quite correct that they're exporting. Thanks for the correction.